### PR TITLE
refactor: move "Sheets._get_paging_lists" to "LazyFactory.create_lists"

### PR DESCRIPTION
**Pull request by Lee-000**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-29xfkd7 将 Sheets._get_paging_lists 迁移至 LazyFactory 内](https://app.clickup.com/t/29xfkd7)

File: graviti/utility/paging.py [hunk 1]
```diff
@@ -721,3 +726,22 @@ def get_page_ranges(self) -> Iterator[range]:
         yield from repeat(range(self._limit), div)
         if mod != 0:
             yield range(mod)
+
+    def get_paging_lists(self, keys: List[Tuple[str, ...]]) -> PagingLists:
```
Comment (linjiX) [inline graviti/utility/paging.py:730]: do we need to input keys?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: graviti/utility/paging.py (hunk 1 above)
Comment (Lee-000) [reply graviti/utility/paging.py:730]: Yes. We don't have `get_keys` for PyArrow type now. It's necessary to get the keys from the Portex schema and pass them into this method.

Comment (linjiX) [reply graviti/utility/paging.py:730]: ok we can implement the get_keys for pyarrow afterwards

Comment (linjiX) [reply graviti/utility/paging.py:730]: `get_paging_lists` -> `create_lists`?

**Review by linjiX — CHANGES_REQUESTED** [review]

**Review by linjiX — APPROVED** [review]

**Review by graczhual — APPROVED** [review]

**Review by AChenQ — APPROVED** [review]

**Pull request merged**